draft-tschofenig-rats-psa-token.md

```diff
@@ -110,9 +110,11 @@ as described in this memo, which are the basis for a number of different
 protocols, including secure provisioning and network access control.  This
 document specifies the PSA attestation token structure and semantics.
 
-At its core, the CWT (COSE Web Token) format is used and populated with a set
-of claims in a way similar to EAT (Entity Attestation Token). This
-specification describes what claims are used by PSA compliant systems.
+The PSA attestation token is a profiled Entity Attestation Token (EAT).
+
+This specification describes what claims are used in an attestation token
+generated by PSA compliant systems, how these claims get serialized to the
+wire, and how they are cryptographically protected.
```

Comment:
Does this have a specific meaning? We don't make any statements about endianess etc

Reply:
We "inherit" the serialisation rules of CBOR.